resume-buddy/src/main/java/com/google/sps/ServletHelpers.java

```diff
@@ -36,6 +44,42 @@ public static boolean collides(UUID id, String entity) {
 
     return results.countEntities(FetchOptions.Builder.withDefaults()) != 0;
   }
+
+  /** Returns a Blobkey that points to the blobstore of the uploaded pdf resume */
+  public static String getBlobstoreKey(
+      HttpServletRequest request, HttpServletResponse response, String formInputElementName) {
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+    Map<String, List<BlobKey>> blobs = blobstoreService.getUploads(request);
+    List<BlobKey> blobKeys = blobs.get(formInputElementName);
+
+    BlobKey blobKey = blobKeys.get(0);
```

Comment:
How do you know to get the first key? When would there be multiple blobkeys?

Reply:
We only have one file input. This function was in RevieweeDataServlet to get form information, I just moved it in ServletHelpers because I wanted to call from different class too. Since reviewee form also has one file input we are only interested in the first key.

Reply:
Got it, thanks for explaining! It would be good to put this in a comment above the line.